src/components/search-filters/filter-group.tsx

```diff
@@ -11,168 +11,184 @@ import theme from '@mdb/flora/theme';
+                                    name === subItem.name &&
+                                    type === subItem.type
+                            )
+                    );
+                    setFilters(filters.concat(filter).concat(subItemsToAdd));
+                } else {
+                    setFilters(filters.concat(filter));
+                }
             } else {
-                setFilters(filters.concat(filter));
+                setFilters(
+                    filters.filter(
+                        ({ name, type }) =>
+                            !(name === filter.name && type === filter.type)
+                    )
+                );
             }
-        } else {
-            setFilters(
-                filters.filter(
-                    ({ name, type }) =>
-                        !(name === filter.name && type === filter.type)
-                )
-            );
-        }
-    };
+        };
 
-    const mobileFilterList = (() => {
-        if (!title) return null;
-        const getFilterNames = (items: FilterItem[]): string[] => {
-            let filterList: string[] = [];
+        const mobileFilterList = (() => {
+            if (!title) return null;
+            const getFilterNames = (items: FilterItem[]): string[] => {
+                let filterList: string[] = [];
 
-            items.forEach(item => {
-                if (filters.includes(item)) {
-                    filterList.push(item.name);
-                }
-                if (item.subItems) {
-                    filterList = filterList.concat(
-                        getFilterNames(item.subItems)
-                    );
-                }
-            });
-            return filterList;
-        };
-        const filterNames = getFilterNames(items);
+                items.forEach(item => {
+                    if (filters.includes(item)) {
+                        filterList.push(item.name);
+                    }
+                    if (item.subItems) {
+                        filterList = filterList.concat(
+                            getFilterNames(item.subItems)
+                        );
+                    }
+                });
+                return filterList;
+            };
+            const filterNames = getFilterNames(items);
 
-        const filterString =
-            filterNames.length < 5
-                ? filterNames.join(', ')
-                : `${filterNames.slice(0, 5).join(', ')} +${
-                      filterNames.length - 5
-                  }`;
-        return (
-            <TypographyScale variant="body1" color="mark">
-                {filterString}
-            </TypographyScale>
-        );
-    })();
+            const filterString =
+                filterNames.length < 5
+                    ? filterNames.join(', ')
+                    : `${filterNames.slice(0, 5).join(', ')} +${
+                          filterNames.length - 5
+                      }`;
```

Comment:
What do you think about moving 5 into a constant, something like `FILTER_STEP`? Then you can use it here and in the next few places (lines 106 - 186).